### PR TITLE
chore: update error message

### PR DESCRIPTION
**Pull request by amanPras**
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Create a variable with length greater than 30 characters. Expect message in below format
`Max length limit of %s characters allowed for %s.`

## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: server/engine/src/main/resources/resourceBundles/messages_en.properties [hunk 1]
```diff
@@ -30,8 +30,8 @@ db2SqlParser.maxIntValue=%s not allowed. Allowed range is 1 to 32767
 db2SqlParser.pieceSize=%s not allowed. Size must be in KB, MB or GB
 db2SqlParser.size=%s not allowed. Size must be in GB
 parsers.validValueMsg=%s not allowed. It must be %s
-parsers.maxLength=Max length limit of %s bytes allowed for %s.
-parsers.exactLength=Exact length of %s must be %s bytes
+parsers.maxLength=Max length limit of %s characters allowed for %s.
```
Comment (zeibura) [inline server/engine/src/main/resources/resourceBundles/messages_en.properties]: Maximum length of %s is 30 characters.
Maximum length of 30 characters for %s.
No need to repeat max, limit and allowed.

File: server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java [hunk 2]
```diff
@@ -175,7 +175,7 @@ void testWhenReplacedLengthIsMoreThanReplaceable() {
             "1",
             new Diagnostic(
                 new Range(),
-                "Max length limit of 30 bytes allowed for variable name.",
+                "Max length limit of 30 characters allowed for variable name.",
```
Comment (zeibura) [inline server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java]: Maximum length of 30 characters for variable names.
Variable names cannot exceed 30 characters.

**Review by zeibura — CHANGES_REQUESTED** [review]

File: server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java (hunk 2 above)
Comment (amanPras) [reply server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java]: updated

File: server/engine/src/main/resources/resourceBundles/messages_en.properties (hunk 1 above)
Comment (amanPras) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties]: updated as per suggestion

File: server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java [hunk 3]
```diff
@@ -175,7 +175,7 @@ void testWhenReplacedLengthIsMoreThanReplaceable() {
             "1",
             new Diagnostic(
                 new Range(),
-                "Max length limit of 30 bytes allowed for variable name.",
+                "variable name cannot exceed 30 characters",
```
Comment (zeibura) [inline server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java]: Capital letter "Variable" (repeat on the other two lines)

**Review by zeibura — CHANGES_REQUESTED** [review]

File: server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java (hunk 3 above)
Comment (amanPras) [reply server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java]: updated

**Review by zeibura — APPROVED** [review]

**Pull request merged**